modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -505,7 +505,19 @@ def test_astype_errors(errors):
     eval_general(modin_df, pandas_df, lambda df: df.astype("int", errors=errors))
 
 
-@pytest.mark.parametrize("has_dtypes", [False, True])
+@pytest.mark.parametrize(
```

Comment:
do we need this change here?

Reply:
these changes are already in master:
https://github.com/modin-project/modin/blob/7195480c9aefa03370a5ff765535bead9c89ad40/modin/pandas/test/dataframe/test_map_metadata.py#L511-L520

@AndreyPavlenko could you please rebase in order to remove them from the diff?